contrib/rpm/podman.spec

```diff
@@ -0,0 +1,207 @@
+# Generate devel rpm
+%global with_devel 1
+# Build with debug info rpm
+%global with_debug 0
+# Run tests in check section
+%global with_check 0
+# Generate unit-test rpm
+%global with_unit_test 1
+
+%if 0%{?with_debug}
+%global _dwz_low_mem_die_limit 0
+%else
+%global debug_package   %{nil}
+%endif
+
+# %if ! 0% {?gobuild:1}
+%define gobuild(o:) go build -tags="$BUILDTAGS selinux seccomp" -ldflags "${LDFLAGS:-} -B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \\n')" -a -v -x %{?**};
+#% endif
+
+%global provider        github
+%global provider_tld    com
+%global project         projectatomic
+%global repo            libpod
+# https://github.com/projectatomic/libpod
+%global provider_prefix %{provider}.%{provider_tld}/%{project}/%{repo}
+%global import_path     %{provider_prefix}
+%global commit         REPLACEWITHCOMMITID
+%global shortcommit     %(c=%{commit}; echo ${c:0:7})
+
+Name:           podman
+Version:        0.1
+Release:        1.git%{shortcommit}%{?dist}
+Summary:        Podman allows you to manage Pods, Containers, and Container Images.
+License:        ASL 2.0
+URL:            https://%{provider_prefix}
+Source0:        https://%{provider_prefix}/archive/%{commit}/%{repo}-%{shortcommit}.tar.gz
+
+# e.g. el6 has ppc64 arch without gcc-go, so EA tag is required
+ExclusiveArch:  %{?go_arches:%{go_arches}}%{!?go_arches:%{ix86} x86_64 aarch64 %{arm}}
+# If go_compiler is not set to 1, there is no virtual provide. Use golang instead.
+BuildRequires:  %{?go_compiler:compiler(go-compiler)}%{!?go_compiler:golang}
+
+
+#dependencies written by hand, not by gofed:
+BuildRequires:  btrfs-progs-devel, device-mapper-devel, glib2-devel, glibc-devel
+BuildRequires:  glibc-static, golang-github-cpuguy83-go-md2man, gpgme-devel, libassuan-devel
+BuildRequires:  libgpg-error-devel, libseccomp-devel, libselinux-devel, ostree-devel
+BuildRequires:  pkgconfig, runc, skopeo-containers
+BuildRequires:  git
+Requires:       runc, skopeo-containers, conmon
+
+
+%description
+%{summary}
+libpod provides a library for applications looking to use the Container Pod concept popularized by Kubernetes.
```

Comment:
OK we say Podman for the name and description above, but nothing here.  Perhaps:

Podman is a command line application that utilizes the libpod library.  The libpod library allows applications to work in a Container Pod concept popularized by Kubernetes.